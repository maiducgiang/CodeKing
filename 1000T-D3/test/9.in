6
9646 6009 9764 8175
4192 9008 6462 603
6951 4324 7893 9043
4958 4050 5399 4106
6453 5428 233 8775
7242 3031 1060 6200
