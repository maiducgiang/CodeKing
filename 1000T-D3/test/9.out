84635243
33648768
59836843
21531592
2768273
10889380
